jwm.md

```diff
@@ -94,4 +94,8 @@ For digital signatures the following algorithms MUST be supported.
 
 ## Signed and Encrypted
 
-When a sender would like for their message to feature both a non-repudiable digital signature and be encrypted to a recipient or multiple recipients. They can combine the encryption and signing operations defined earlier in this section by creating a [nested JWM](https://tools.ietf.org/html/draft-looker-jwm-01#section-1.2).
\ No newline at end of file
+When a sender would like for their message to feature both a non-repudiable digital signature and be encrypted to a recipient or multiple recipients. They can combine the encryption and signing operations defined earlier in this section by creating a [nested JWM](https://tools.ietf.org/html/draft-looker-jwm-01#section-1.2).
+
+## Message Authorization
+
+In order to validate the authenticity of the message, the recipient MUST validate that the keys used by the sender in the JWE and/or JWS are valid and within the did document of the expected decentralized identifier. To verify the authorization of the message, the recipient MUST resolve a valid DID Document associated with the DID in the `from` field of the JWM message. The recipient MUST then validate that the keys and key ids (kid) used in the JWE/JWS match keys and identifiers associated (via the did document) with the decentralized identifier. The recipient also MUST verify that a key used for [Sender Authenticated Encryption](#Sender-Authenticated-Encryption) is associated with the verification relationship property named `keyAgreement` in order for the message to be authorized. In the case of a [signed](#Signing) message, the key must maintain a verification relationship with the `assertionMethod` property in order for the message to be authorized. In the case of a [nested](Signed-and-Encrypted) message the associated keys and key ids MUST be valid for the message to be considered authorized. If the keys are not contained within the did document then the message should not be considered authorized, even if the cryptographic operations succeeds.
```

Comment:
`assertionMethod` is for creating VCs... I would expect `authentication` in this case.